deploy-deps.sh

```diff
@@ -48,13 +48,15 @@ deploy_tekton() {
     # Operator
     kubectl apply -k "${script_path}/dependencies/tekton-operator"
     retry kubectl wait --for=condition=Ready -l app=tekton-operator -n tekton-operator pod --timeout=240s
+    # Wait for the operator to create configs for the first time before applying configs
+    kubectl wait --for=condition=Ready TektonConfig/config --timeout=360s
```

Comment:
```suggestion
    kubectl wait --for=condition=Ready tektonconfig/config --timeout=360s
```
for consistency with the second call

Reply:
done